src/modules/admin/components/AdminDashboard/AdminDashboard.tsx

```diff
@@ -93,8 +93,8 @@ const navigationItems = (colony: ColonyType): NavigationItem[] => [
   },
   {
     id: 5,
-    title: MSG.tabTransaction,
-    content: <Transactions colonyAddress={colony.colonyAddress} />,
+    title: MSG.tabAdvanced,
+    content: <ProfileAdvanced colony={colony} />,
```

Comment:
Since `ProfileAdvanced` no longer has tabs at the top, I think it's deserving of a heading there (just like `ProfileEdit`). This isn't spec'd in figma, though, so maybe get with product to confirm

Reply:
That's a bit strange, maybe I committed after you reviewed, every tab has a heading by now. What I will confirm with product though is if they want me to adjust their layout a bit since they are currently jumping a bit.